config/config.public.example.json

```diff
@@ -1,4 +1,4 @@
 {
     "client_login_url": "http://localhost:3003/submit",
-    "client_api_url": "http://localhost:9000"
+    "client_api_url": "http://localhost:3003"
 }
```

Comment:
This file can be removed entirely, right?

Reply:
removed